desktop-sync/latest/admin/index.md

```diff
@@ -1,7 +1,9 @@
 ---
-title: Administration of Desktop Sync
+title: Administer Desktop Sync
 ---
 
+If you’re an IT administrator, you can configure a number of features to centrally manage your Desktop Sync clients.
```

Comment:
```suggestion
If you’re an I.T. administrator, you can configure a number of features to centrally manage your Desktop Sync clients.
```

Reply:
Just trying out the suggestion functionality. Should it be with full stops I dont know?

Reply:
In general both versions are acceptable, but common usage in our docs is without the full stops.